homeassistant/components/fitbit/const.py

```diff
@@ -65,3 +65,18 @@ class FitbitUnitSystem(StrEnum):
 
     EN_GB = "en_GB"
     """Use United Kingdom units."""
+
+
+OAUTH2_AUTHORIZE = "https://www.fitbit.com/oauth2/authorize"
+OAUTH2_TOKEN = "https://api.fitbit.com/oauth2/token"
+OAUTH_SCOPES = [
+    "activity",
+    "heartrate",
+    "nutrition",
+    "profile",
+    "settings",
+    "sleep",
+    "weight",
+]
+# Authentication domain for credentials imported from the legacy fitbit.conf
+AUTH_IMPL_IMPORTED = "fitbit-conf-imported"
```

Comment:
I thought it would be useful to be able to disambiguate in case of a bug in the future e.g. import fails in some cases and we need to patch the existing token somehow. I don't have a concrete use case right now, so this is being defensive against a future problem.

Happy to reconsider if this is a "YAGN" case.

Reply:
what do you think about a separate key in the data? "imported":" true or something. If you ever add diagnostics you can use that to track it down. I think the auth implementation would be a big pain to change later on (for whatever reason the future may behold)

Reply:
Ah, i realize the other part is that the new config entry needs to refer to this. However, the default is just DOMAIN so thats fine.  I've reverted this so it is now set to the default of DOMAIN which is also identifiable since UI entered application credentials will have a unique id based on the client id.

Also, i realize some of these yaml parameters are always set and not used when configured from the UI so that should be fine (e.g. 'MONITORED_RESOURCES' being present can serve as `"imported": True`.

Let me know if that all seems good.